rasa/utils/tensorflow/model_data.py

```diff
@@ -1135,8 +1167,8 @@ def _4d_scipy_matrix_to_values(array_of_array_of_sparse: FeatureArray):
                 for array_of_sparse in array_of_array_of_sparse
             ]
 
-        max_dialogue_len = max(
-            [len(array_of_sparse) for array_of_sparse in array_of_array_of_sparse]
+        max_dialogue_len = sum(
```

Comment:
it is not a `max_dialogue_len`, right? I think smth like `combined_dialogue_len` would make more sense here